qiita_pet/handlers/study_handlers/tests/test_artifact.py

```diff
@@ -140,6 +140,33 @@ def test_delete_artifact(self):
         wait_for_prep_information_job(1)
 
 
+class ArtifactGetSamplesTest(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/artifact/samples/', {'ids[]': [4, 5]})
+        self.assertEqual(response.code, 200)
+        exp = (
+            '{"status": "success", "msg": "", "data": {"4": ['
```

Comment:
I would recommend changing it, the fact that we haven't seen this fail yet doesn't mean that it can't fail as the assumption it relies on is not guaranteed to be consistent.